### PR TITLE
Chow NI tomography integration code

### PR DESCRIPTION
**Pull request by sungeunbae**
Chow NI tomography (https://core.geo.vuw.ac.nz/d/feae69f61ea54f81bee1/ ) integrated

### CODE REVIEW

**Comment by jasonmotha** [comment]
Github won't let me view the changes on this commit [c580cfa](https://github.com/ucgmsim/Velocity-Model/pull/34/commits/c580cfa408cf131a467f154c3f5f402ac12464bd) I believe it's more than just the files added. Can you add these files in as a separate PR and then the the code changes as a separate?

Regarding the actual VM domain merge code, I think it would be good to have it in this repo too. There are some python scripts in the Velocity Model folder in here.

**Comment by sungeunbae** [comment]
> Github won't let me view the changes on this commit [c580cfa](https://github.com/ucgmsim/Velocity-Model/pull/34/commits/c580cfa408cf131a467f154c3f5f402ac12464bd) I believe it's more than just the files added. Can you add these files in as a separate PR and then the the code changes as a separate?
> 
> Regarding the actual VM domain merge code, I think it would be good to have it in this repo too. There are some python scripts in the Velocity Model folder/module in here.

Looks like github did some magic when it complained my PR was outdated.

**Comment by sungeunbae** [comment]
Review this after merging https://github.com/ucgmsim/Velocity-Model/pull/39
Screen Shot 2022-03-14 at 10 56 56 PM

**Comment by sungeunbae** [comment]
Screen Shot 2022-03-17 at 6 23 56 AM


@jasonmotha @jpa198  This should be more reviewer friendly now.

**Comment by jasonmotha** [comment]
It's much better now. Cheers.

Can we model the version numbers for 2.08 similarly to how Ethan upgraded the tomography for 2.05?

2.08 for the version (with all the 2.07 basins) sorry there's been a few more added since you started looking at this.
2.08_no_basins - for the version with no-basins

File: src/makefile.mac [hunk 1]
```diff
@@ -1,7 +1,7 @@
 TARGET = NZVM
-LIBS = -lm
-CC = gcc-8
-CFLAGS = -std=c99 -g -Wall -O3 -fopenmp #-D_AIX
```
[the remark this answers is not in the document]
Comment (sungeunbae) [reply src/makefile.mac:4]: OK, to clarify, are you suggesting 2.08_no_basins (currently named 2.08_chow_ni) and 2.08 (currently named 2.09_chow_ni)? I am happy to change.

[the remark this answers is not in the document]
Comment (sungeunbae) [reply src/makefile.mac:4]: and for the makefile change, yeah..I admit it is irrelevant from this PR, but was needed to compile the code on my mac. Then, I wondered if it was worth a separate PR, and decided to let it go under your eyes..haha.

[the remark this answers is not in the document]
Comment (jasonmotha) [reply src/makefile.mac:4]: > OK, to clarify, are you suggesting 2.08_no_basins (currently named 2.08_chow_ni) and 2.08 (currently named 2.09_chow_ni)? I am happy to change.

Yep. That sounds right.

[the remark this answers is not in the document]
Comment (sungeunbae) [reply src/makefile.mac]: Cool. Just pushed the change. Will make a separate makefile.mac PR.

**Review by jasonmotha — APPROVED** [review]
Just had a look at the whole makefile situation. (it's a bit simpler that I remember).

You're probably the only person who'll use the make mac option going forward; so I don't think it's a big issue with your changes.

**Review by jasonmotha — CHANGES_REQUESTED** [review]
Oh, can you include the additional basins that have snuck into 2.07 ?

**Comment by sungeunbae** [comment]
> Oh, can you include the additional basins that have snuck into 2.07 ?

Done

**Review by jasonmotha — APPROVED** [review]

**Pull request merged**